Source/WebKit/_WebKit_SwiftUI/API/View+WebViewModifiers.swift

```diff
@@ -101,7 +100,15 @@ extension View {
 #endif
     }
 
-    @_spi(Private)
+    /// Specifies the visibility of the webpage's natural background color within this view.
+    ///
+    /// By default, WebViews are opaque, and use the page's natural background color as their background color. Use this modifier if you would like to
+    /// not use this behavior and instead provide a custom background using SwiftUI.
+    ///
+    /// - Parameter visibility: The visibility to use for the background.
```

Comment:
Above we have `Parameters` plural with a single bullet. What is the right formatting?

Reply:
both are technically valid, but I think when there is only one parameter this way is the more correct one